gm4_block_compressors/data/gm4_block_compressors/functions/process.mcfunction

```diff
@@ -1,6 +1,8 @@
 # @s = block compressor
 # run from main
+tag @s add gm4_block_compressor_processing
 execute positioned ~ ~1 ~ align xyz as @e[type=item,dx=0] positioned ~0.5 ~-0.9 ~0.5 run function gm4_block_compressors:item
+tag @s remove gm4_block_compressor_processing
 
 particle portal ~ ~1.1 ~ 0 0 0 0.1 20
 data merge entity @s {Fire:2000}
```

Comment:
Strangely, Github is not showing any conflicts from this line, though [this commit](https://github.com/Gamemode4Dev/GM4_Datapacks/commit/b83816d695326a3589ddb749cc48249c49157a23) should create one. 

Unless my Github is just acting odd, this should be changed to use `HasVisualFire`

Reply:
I believe there’s no conflicts because this isn’t changing any of those lines, it’s simply adding lines before and after line 3, which the PR you linked to doesn’t touch. If it was the other way around it might cause a conflict (i.e. if this one was merged first, then the visual fire PR was created, then the lines it’s changing would be different)

Reply:
Oh excellent look at git working the way it should :tada: